clicky/clickyScript.py

```diff
@@ -70,8 +76,8 @@ def plotData(allData):
     plt.plot(X1, Y1, 'o-', label = 'Total Visits') 
```

Comment:
It just occurred to me that "Total Visits" could be a bit misleading / vague. I think it would help to make this data label 'Total Weekly Visits'. Adding "weekly" to the Y-axis label and plot title might help as well.

Reply:
Thanks!